src/translations/locales/common_sa-SA.json

```diff
@@ -0,0 +1,91 @@
+{
+  "core": {
+    "title": "Vaxx.nz",
+    "tagline": "Avanoa o Tuipuipui ole COVID-19",
+    "subtitle": "Tagai i auala e faia ai tuipuipui i nofoaga latalata ane ia te oe.",
+    "disclaimerNotAGovWebsite": "O lenei 'upega tafa'ilagi e lei faia ele Malo. Ile faatinoina o tuipuipui asiasi ane ile 'upega tafa'ilagi lea, <0>bookmyvaccine.nz</0>",
+    "loading": "utaina",
+    "getDirections": "Vaai fa'atonuga",
+    "makeABooking": "Faamauina lou avanoa",
+    "kmAway": "{{distance}} kilomita le alu",
+    "lastUpdated": "Taimi mulimuli ga fa'afouina ai - {{updatedAt}}",
+    "cancel": "Fa'aleaogaina",
+    "website": "'Upega tafa'ilagi"
+  },
+  "navigation": {
+    "about": "About",
+    "contact": "Contact",
+    "getInvolved": "Get Involved",
```

Comment:
Do we want to translate these also?

Reply:
@tausani-ah-chong if there is a translation for it that you would typically see on websites than yes please! Let me know when you're all done